src/directives/ec-amount/ec-amount.spec.js

```diff
@@ -48,17 +47,17 @@ describe('EcAmount', () => {
     const { wrapper, input } = mountTemplate('<input v-ec-amount data-test="amount-input" />');
     const spy = jest.spyOn(input.element, 'removeEventListener');
 
+    const inputHandler = input.element.__inputHandler;
+    const keydownHandler = input.element.__keydownHandler;
+
     wrapper.unmount();
 
-    expect(spy).toHaveBeenCalledTimes(2);
+    expect(spy.mock.calls).toContainEqual(['input', inputHandler]);
+    expect(spy.mock.calls).toContainEqual(['keydown', keydownHandler]);
   });
 
   it('should throw an error if the directive is not attached to an input', () => {
-    withMockedConsole((errorSpy, warnSpy) => {
-      expect(() => mountTemplate('<div v-ec-amount="{}"></div>')).toThrow(new TypeError('v-ec-amount requires 1 input'));
-      expect(warnSpy).toHaveBeenCalledTimes(1);
-      expect(warnSpy.mock.calls[0][0]).toContain('Unhandled error during execution of directive hook');
-    });
+    expect(() => mountTemplate('<div v-ec-amount="{}"></div>')).toThrow(new TypeError('v-ec-amount requires 1 input'));
```

Comment:
Out of curiosity, is this mentioned somewhere in the changelog why this has changed? is this a change in VTU or in Vue?

Reply:
I couldn´t find anything about that, but I would say is a change in Vue itself because it only happens with those unhandled errors, other errors such as [this one](https://github.com/Ebury/chameleon/pull/1579/commits/d1cdf3e03c59771fe4808f506df1aadf733e2674#diff-990b8ff5535146e45bbe39346c6918d91dd1d03380f7fbf22744bf55f4e784b5L160) are being printed in the console, so VTU is not really involved on that